### PR TITLE
New Port: Grimstorm

### PR DESCRIPTION
**Pull request by JanTrueno**
# New Port for Grimstorm

## Game Information
- **Title**: Grimstorm
- **URL**: https://gamejolt.com/games/grimstorm/59563 

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/grimstorm/port.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "inst": "Download the windows version of the game here: https://gamejolt.com/games/grimstorm/59563\nPlace Grimstorm1.3.2.exe inside the gamedata folder.",
+    "genres": [
+      "action"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [
```
Comment (kloptops) [inline ports/grimstorm/port.json:24]: `gmloader` and `7zzs` are both `armhf` but you have flagged the port as `aarch64`.

**Review by kloptops — CHANGES_REQUESTED** [review]
Wrong arch in port.json

**Review by Cebion — APPROVED** [review]
👍

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**